website/docs/tutorials/app-demo-tutorial.md

```diff
@@ -0,0 +1,566 @@
+It also showcases how quickly and effortlessly a feature can be rolled back if it does not behave as expected.
+***
+
+### Special Customer
+
+In the UI, navigate to *Management -> E-Commerce -> Shop*. Enable the toggle for feature `demo-specialCustomer` . 
+Notice how cards are updated with *Special Access Granted* and a *Premium Customer* note appears at the top. <!-- Add Screenshot -->
+
+![demo-specialCustomer](/img/tutorial-appdemo-demo-specialCustomer.png)
+
+***tip Use Case
+This shows how a feature flag can be used to show a different page experience or behavior to a specific group of users; in this case, a group of customers that we want to allow the capability to purchase a product standard users cannot.
+***
+
+### Analytics Cards
+
+Navigate to *General -> Analytics* from the main demo app window. Enable the toggle for feature `demo-analyticsCards` . Note how a row of analytics cards now shows from above. 
+
+![demo-analyticsCards](/img/tutorial-appdemo-demo-analyticsCards.png)
+
+
+***tip Use Case
+This shows how a feature flag can be used to show a different page experience or behavior to a specific group of users; in this case, displaying a set of cards that might be used by specific user roles that need to report on analytics data for the website.
+***
+
+## Server-side Feature Flags
+
+### Featured App
+
+Navigate to *General -> App* from the main demo app window. Enable the toggle for feature `demo-featured_app` and reload the page using the browser controls.
+
+***tip Reload
+In contrast to the client side flags, this one requires you to reload the browser of the page to see the difference. This is because the toggle operates on the backend of the web app, therefore the client needs to retrieve the page again to see it.
+***
+
+Notice that we now see a *Featured App* card appear on the UI.
+
+![demo-featured_app](/img/tutorial-appdemo-demo-featured_app.png)
+
+Disable it again, and it will no longer show on the next reload.
+
+
+# SDK Integration - A Closer Look
+
+Now that we've seen the feature toggles and their underlying use cases in action, let's investigate how they operate behind the scenes in the source code.
+The key objective of this final topic is to help understand how you would implement this into your own application.
+
+We start with the Unleash [https://docs.getunleash.io/reference/sdks/next-js](Next.js SDK documentation) - open up this page where we'll review the Installation and Environment variables.
+
+First of all, to prepare the environment, the SDK will be installed from the package manager of choice (npm, yarn, pnpm).  
+
+Now see the section on `Environment Variables` - observe how these match the contents of our `docker-compose.yml` which sets these up inside the application container.
+
+Then, there are specific steps for the Client side and Server side use cases. Let's investigate each of these.
+
+
+## Client SDK
```

Comment:
Because the SDK integration heading gets demoted, so do the following subsections

```suggestion
### Client SDK
```